src/auth/email.service.ts

```
@@ -0,0 +1,35 @@
+      from: process.env.EMAIL_USER,
+      to: options.to,
+      subject: options.subject,
+      text: options.text,
+    };
+
+    try {
+      await this.transporter.sendMail(mailOptions);
+     
+    } catch (error) {
+      
+      throw error; 
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)